web/controllers/auth.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Notox.Auth do
+  import Plug.Conn
+
+  def init(_opts) do
+  end
+
+  defp get_current_user(conn) do
+    current_user_id = get_session(conn, :user_id)
```

Comment:
We'd discussed this previously, but I'd be precise about which user you're talking about, so `current_user_id`.